dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -90,7 +92,10 @@ def leave
 
   def update_sharing_disabled
     @section.update!(sharing_disabled: params[:sharing_disabled])
-    render json: {sharing_disabled: @section.sharing_disabled}
+    render json: {
+      sharing_disabled: @section.sharing_disabled,
+      students: @section.students
```

Comment:
Do we really need to be passing the whole `User` model for students (including PII and such) in the JSON return?

Reply:
nope, good point

Reply:
Actually, we do need a lot of it given that this will populate the Manage Student table, but I'll use the summarize function.